admin-dev/themes/new-theme/scss/pages/_modules.scss

```diff
@@ -569,22 +569,98 @@ form#importDropzone {
     display: table;
     width: 100%;
     padding: 5px 0;
-    margin-bottom: -1px;
-    border: solid 1px #9ed0ec;
+    // margin-bottom: -1px;
+    border-bottom: solid 1px #bbcdd2;
     background: #fff;
+    &:hover{
+      background-color: #fafbfc;
 
+      .drag_indicator{
+        color: #b3c7cd;
+        opacity: 1;
+      }
+    }
+    &:focus{
+      outline: none;
+    }
+    &:active,&:visited{
```

Comment:
These two should be in a line of their own. Also, please mind the indentation and spacing around braces.

Reply:
I fixed it